src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &-star {
+    background: center / no-repeat url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &-star:last-child {
+    margin-right: 0;
+  }
+
+  &--1 &-star:nth-child(-n + 1),
+  &--2 &-star:nth-child(-n + 2),
+  &--3 &-star:nth-child(-n + 3),
+  &--4 &-star:nth-child(-n + 4),
+  &--5 &-star:nth-child(-n + 5) {
```

Comment:
There is some repetition in your code. You have written almost the same code for each of the modifier classes. Consider using a loop to generate these classes. This would make your code DRY (Don't Repeat Yourself), which is a good practice in programming. In SCSS, you can use the `@for` directive to generate repetitive CSS.